<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(21, 130, 213);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnVender">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string> VENDER</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../IMG/vender.png</normaloff>../IMG/vender.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProductos">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string> PRODUCTOS</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../IMG/producto.png</normaloff>../IMG/producto.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClientes">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string> CLIENTES</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../IMG/user.png</normaloff>../IMG/user.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAlmacenes">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string> ALMACENES</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../IMG/almacen.png</normaloff>../IMG/almacen.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProveedores">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string> PROVEEDORES</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../IMG/proveedor.png</normaloff>../IMG/proveedor.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalidas">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string> SALIDAS</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../IMG/ingreso y salida.png</normaloff>../IMG/ingreso y salida.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVentas">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string> VENTAS</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../IMG/ingreso y salida.png</normaloff>../IMG/ingreso y salida.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDetalleVentas">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string> DETALLE VENTAS</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../IMG/punto-de-venta.png</normaloff>../IMG/punto-de-venta.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalir">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>  SALIR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../IMG/letra-x (1).png</normaloff>../IMG/letra-x (1).png</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>211</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 255);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../IMG/edusis.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>551</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(23, 143, 234);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>btnVender</zorder>
   <zorder>btnProductos</zorder>
   <zorder>label_2</zorder>
   <zorder>btnClientes</zorder>
   <zorder>btnAlmacenes</zorder>
   <zorder>btnProveedores</zorder>
   <zorder>btnSalidas</zorder>
   <zorder>btnVentas</zorder>
   <zorder>btnDetalleVentas</zorder>
   <zorder>label</zorder>
   <zorder>btnSalir</zorder>
   <zorder>label_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
